tests/functional/lti_certification/test_lti_certification_1_1.py

```diff
@@ -0,0 +1,99 @@
+import time
+
+import oauthlib.common
+import oauthlib.oauth1
+import pytest
+from h_matchers import Any
+
+from lms.models import ApplicationInstance
+from tests.functional.base_class import TestBaseClass
+
+
```

Comment:
If there are any smallish changes that need to be made here, you might want to hold off until the next ticket, as it rewrites a decent portion here.